doc/classes/PhysicsServer3D.xml

```diff
@@ -1066,6 +1081,14 @@
 				Sets the value for a space parameter. A list of available parameters is on the [enum SpaceParameter] constants.
 			</description>
 		</method>
+		<method name="space_step">
+			<return type="void" />
+			<param index="0" name="space" type="RID" />
+			<param index="1" name="delta" type="float" />
+			<description>
+				Step a specific space. Is usaful to create a rollbackable simulation. You should call [method space_flush_queries] first.
```

Comment:
```suggestion
				Steps [param space]. This can be used to create a rollbackable simulation. You should call [method space_flush_queries] first.
```

Reply:
I would avoid punning parameters when possible. There is hardly any confusion for functions that have only one or two parameters. But punning makes the life of translators hard.